ports/intel-mkl/portfile.cmake

```
@@ -237,6 +237,12 @@ endif()
 file(COPY "${mkl_dir}/lib/cmake/" DESTINATION "${CURRENT_PACKAGES_DIR}/share/")
 vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/mkl/MKLConfig.cmake" "MKL_CMAKE_PATH}/../../../" "MKL_CMAKE_PATH}/../../")
 vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/mkl/MKLConfig.cmake" "redist/\${MKL_ARCH}" "bin")
+if(${VCPKG_LIBRARY_LINKAGE} STREQUAL "static")

```

```suggestion
if("${VCPKG_LIBRARY_LINKAGE}" STREQUAL "static")
```

Not worth rebuilding over this. Maybe something next time this file is edited.

---

OK